### PR TITLE
fix(admin): pagination set currentPage by switch tabs

### LINKED ISSUE #2818: 🐛 [Bug][Admin] Pagination is not reset when switching tabs
## 🐛 Bugreport

On the page Contribution when I go on the tab "Deleted" and select page 20, and switching on tab "All" we still stay on page 20. 
If I go on the last page and go back to another tab with less Contribution pages we are set on page 1.

## Todo
- [ ] Reset pagination/page "index" when changing tab

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
there is a watc h on tabIndex, if this changes then currentPage is set to 1

### Issues
- fixes #2818

### CODE REVIEW

**Review by Elweyn — APPROVED** [review]

**Review by Mogge — COMMENTED** [review]
Where does the cypress update come from?

File: admin/src/pages/CreationConfirm.vue [hunk 1]
```diff
@@ -116,6 +116,11 @@ export default {
       pageSize: 25,
     }
   },
+  watch: {
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.vue:119]: Can we please have a unit test for at least one case?

Comment (Elweyn) [reply admin/src/pages/CreationConfirm.vue:119]: [c6bde11](https://github.com/gradido/gradido/pull/2902/commits/c6bde11de064eafcfa171e5a9198cdd3eb8edefa)

File: admin/src/pages/CreationConfirm.spec.js [hunk 2]
```diff
@@ -152,6 +152,35 @@ describe('CreationConfirm', () => {
...
+          expect(wrapper.findComponent({ name: 'BPagination' }).exists()).toBe(true)
+        })
+
+        describe('next page', () => {
+          beforeAll(async () => {
+            jest.clearAllMocks()
+          })
+
+          beforeEach(() => {
+            wrapper.vm.currentPage = 2
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:166]: Please emit  input from the pagination component

File: admin/src/pages/CreationConfirm.spec.js [hunk 3]
```diff
@@ -367,6 +396,10 @@ describe('CreationConfirm', () => {
             await wrapper.find('a[data-test="denied"]').trigger('click')
           })
 
+          it('refresh the currentPage to 1', () => {
+            expect(wrapper.vm.currentPage).toBe(1)
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:400]: Is 1 by default

File: admin/src/pages/CreationConfirm.spec.js [hunk 4]
```diff
@@ -383,6 +416,10 @@ describe('CreationConfirm', () => {
             await wrapper.find('a[data-test="deleted"]').trigger('click')
           })
 
+          it('refresh the currentPage to 1', () => {
+            expect(wrapper.vm.currentPage).toBe(1)
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:420]: default

File: admin/src/pages/CreationConfirm.spec.js [hunk 5]
```diff
@@ -399,6 +436,10 @@ describe('CreationConfirm', () => {
             await wrapper.find('a[data-test="all"]').trigger('click')
           })
 
+          it('refresh the currentPage to 1', () => {
+            expect(wrapper.vm.currentPage).toBe(1)
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:440]: default

File: admin/src/pages/CreationConfirm.spec.js [hunk 6]
```diff
@@ -399,6 +436,10 @@ describe('CreationConfirm', () => {
...
+          })
+
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:442]: After the next it block, add a new describe block and test the pagination there (we have most items in all tab). Then, in another describe, click back to the open tab to check, that currentPage is really reseted

File: admin/src/pages/CreationConfirm.spec.js [hunk 7]
```diff
@@ -152,6 +152,35 @@ describe('CreationConfirm', () => {
     })
 
     describe('actions in overlay', () => {
+      describe('change pagination', () => {
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.spec.js:155]: Move this block after inside the all tab action

**Review by Mogge — CHANGES_REQUESTED** [review]

**Review by mahula — APPROVED** [review]

**Review by Mogge — APPROVED** [review]

**Pull request merged**